tests/models/univnet/test_modeling_univnet.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import inspect
+import random
+import unittest
+
+from datasets import Audio, load_dataset
+
+from transformers import UnivNetGanConfig, UnivNetGanFeatureExtractor
+from transformers.testing_utils import (
+    is_torch_available,
+    require_torch,
+    require_torch_gpu,
+    slow,
+    torch_device,
+)
+
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import (
+    ModelTesterMixin,
+    floats_tensor,
+)
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import UnivNetGan
+
+
+class UnivNetGanTester:
+    def __init__(
+        self,
+        parent,
+        batch_size=13,
```

Comment:
Also here let's keep the batch size small for a fast model test
```suggestion
        batch_size=2,
```